static/src/stylesheets/module/commercial/glabs/_hosted-gallery.scss

```diff
@@ -548,12 +548,8 @@ $progressColor: #ffffff;
     width: $progressHeight;
 
     &:before {
-        content: "";
+        content: '';
         position: absolute;
-        background-image: -o-radial-gradient(center center, ellipse cover, $neutral-1 0%, transparent 50%);
```

Comment:
@lps88 I have removed these as I think auto prefixer deals with this anyway. Is that okay with you?